akd/src/tests.rs

```diff
@@ -354,7 +385,8 @@ async fn test_simple_audit() -> Result<(), AkdError> {
     Ok(())
 }
 
-#[tokio::test]
+// #[tokio::test]
+#[allow(dead_code)]
```

Comment:
nit: can we re-enable all `dead_code` tests?

Reply:
This is the only test that is disabled. We discussed offline that this should be addressed in another PR, likely, since this failure is due to not storing hashes from the current completed epoch while updating. See https://github.com/novifinancial/akd/issues/203